### PR TITLE
MHV-55471: Downtime notification updates complete

### PR DESCRIPTION
**Pull request by mattwrightva**
## Summary
- The downtime notification messaging has been updated to match the most recent content provided by the UCD and content teams. 

## Related issue(s)
https://jira.devops.va.gov/browse/MHV-55471
Screenshot 2024-03-11 at 8 01 32 AM
Screenshot 2024-03-11 at 8 02 02 AM
Screenshot 2024-03-11 at 8 01 53 AM
Screenshot 2024-03-11 at 8 02 14 AM

## Screenshots
Downtime approaching:
Screenshot 2024-03-11 at 8 49 21 AM
Screenshot 2024-03-11 at 8 48 31 AM

Downtime active:
Screenshot 2024-03-11 at 7 30 01 AM
Screenshot 2024-03-11 at 8 06 25 AM

## What areas of the site does it impact?
- Platform downtime notifications
- MHV Medical records
- MHV Secure messaing

## Acceptance criteria
Downtime notification content has been updated

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

File: src/platform/mhv/downtime/components/MHVDowntimeApproaching.jsx [hunk 1]
```diff
@@ -13,8 +13,8 @@ function MHVDowntimeApproaching({
         <h2 slot="headline">Upcoming maintenance on My HealtheVet</h2>
         <p>
           We&#x2019;ll be working on My HealtheVet soon. The maintenance will
-          last {timeInterval}. During this time, you may have trouble using{' '}
-          {appTitle}.
+          last {timeInterval}. During this time, won't be able to use this{' '}
```
Comment (oleksii-morgun-va) [inline src/platform/mhv/downtime/components/MHVDowntimeApproaching.jsx:16]: Is it supposed to be `you won't be able to use this` ? 
```suggestion
          last {timeInterval}. During this time, you won't be able to use this{' '}
```

**Review by alex-morgun-1 — DISMISSED** [review]
great 👍🏻

**Review by mmoyer-va — DISMISSED** [review]

**Review by KyleCardwell — DISMISSED** [review]
LGTM!

**Review by dcloud — CHANGES_REQUESTED** [review]
Update per https://dsva.slack.com/archives/C04DRS3L9NV/p1710177936773819?thread_ts=1709757868.565759&cid=C04DRS3L9NV:

> … ok the content is now updated in the Mural section it should be "During this time, you won't be able to use our health tools."

~~The changes here remove the "you may have trouble" versions of the text in the MHVDowntime components, but that text is still needed.~~

I think we need a param to choose which text we want to use. If the tool-specific text is the new default, then `src/applications/mhv-landing-page` will need updating to use the "you may have trouble" language

**Comment by mattwrightva** [comment]
> The changes here remove the "you may have trouble" versions of the text in the MHVDowntime components, but that text is still needed.
> 
> I think we need a param to choose which text we want to use. If the tool-specific text is the new default, then `src/applications/mhv-landing-page` will need updating to use the "you may have trouble" language

This should no longer be needed

**Review by dcloud — APPROVED** [review]
I think this looks good. I noted that the Mural has a slight inconsistency on whether an appTitle of "My HealtheVet" is used, but your PR corrects that inconsistency. Thanks for figuring this out.

**Review by alex-morgun-1 — APPROVED** [review]
👍🏻 re-approving

**Review by mmoyer-va — APPROVED** [review]

File: src/platform/mhv/downtime/components/MHVDown.jsx [hunk 2]
```diff
@@ -12,9 +12,8 @@ function MHVDown({
       <va-alert class="vads-u-margin-bottom--4" status="error" uswds visible>
         <h2 slot="headline">Maintenance on My HealtheVet</h2>
         <p>
-          We&#x2019;re working on My HealtheVet. The maintenance will last{' '}
-          {timeInterval}. During this time, you may have trouble using{' '}
-          {appTitle}.
+          We&#x2019;re working on {appTitle} right now. The maintenance will
```
[the remark this answers is not in the document]
Comment (mattwrightva) [reply src/platform/mhv/downtime/components/MHVDown.jsx:15]: I agree, I'll double check with UCD just to be thorough...

**Review by ataker — APPROVED** [review]

**Pull request merged**